db/migrate/20160324174218_create_container_deployment_nodes.rb

```diff
@@ -0,0 +1,17 @@
+class CreateContainerDeploymentNodes < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployment_nodes do |t|
+      t.string :ip_or_hostname
+      t.string :name
+      t.text :labels
+      t.belongs_to :container_deployment, :type => :bigint
+      t.belongs_to :vm,                   :type => :bigint
+      t.text :docker_storage_devs, :array => true, :default => []
+      t.string :docker_storage_data_size
+      t.text :customize
+      t.boolean :deploy_metrics, :default => true
+      t.boolean :deploy_registry, :default => true
```

Comment:
@simon3z Are you expecting to use them in a CloudForms follow up feature for this release?  If so, I'd let it in, but if not (i.e. if you're expecting to use it in the e-release), then I wouldn't put it in this PR.